crates/sql-sink/src/main.rs

```diff
@@ -27,49 +29,75 @@ const BACKOFF_MAX: Duration = Duration::from_secs(3600 * 24);
 
 #[connector(sink)]
 async fn start(config: SqlConfig, mut stream: impl ConsumerStream) -> Result<()> {
-    let mut sink = start_sink(config).await?;
-    while let Some(item) = stream.next().await {
-        let operation: Operation = serde_json::from_slice(item?.as_ref())?;
-        trace!(?operation);
-        sink.send(operation).await?;
+    let mut backoff = backoff_init()?;
+    let mut sink = start_sink(&mut backoff, &config).await?;
+
+    info!("Starting to process records");
+
+    while let Some(item_result) = stream.next().await {
+        match item_result {
+            Ok(item) => match serde_json::from_slice::<Operation>(item.as_ref()) {
+                Ok(operation) => {
+                    trace!(?operation);
+                    while let Err(e) = sink.send(operation.clone()).await {
+                        error!("Error sending operation to sink: {}", e);
```

Comment:
Error message should be more clear. Something like "error trying to send to db"

Reply:
fixed!